app-editors/vim/vim-8.2.3428.ebuild

```
@@ -0,0 +1,329 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+VIM_VERSION="8.2"
+LUA_COMPAT=( lua5-1 luajit )
+PYTHON_COMPAT=( python3_{7..10} )
```

Does this need updating? Or do we disable python for vim?
```suggestion
PYTHON_COMPAT=( python3_{6..10} )
```

---

python is already disabled for both SDK and /build/...

```
$ emerge --pretend --verbose app-editors/vim
[ebuild   R   ~] app-editors/vim-8.2.3428::portage-stable  USE="acl nls -X -cscope -debug -gpm -lua -minimal -perl -python -racket -ruby -selinux -sound -tcl -terminal -vim-pager" LUA_SINGLE_TARGET="lua5-1 -luajit" PYTHON_SINGLE_TARGET="(-python3_10) (-python3_7) (-python3_8) (-python3_9)" PYTHON_TARGETS="(-python3_10) (-python3_7) (-python3_8) (-python3_9)" 0 KiB
```